src/migration/1721301769783-PostStatsUpdated.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class PostStatsUpdated1721301769783 implements MigrationInterface {
+    name = 'PostStatsUpdated1721301769783'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "post" ADD "statsUpdatedAt" TIMESTAMP NOT NULL DEFAULT now()`);
+
+        await queryRunner.query(`
+          CREATE OR REPLACE FUNCTION post_stats_updated_at_time()
+            RETURNS TRIGGER
+            LANGUAGE PLPGSQL
+            AS
+          $$
+          BEGIN
+            NEW."statsUpdatedAt" = now();
```

Comment:
Ah for some reason I assumed we'd move the whole logic of plussing the actual numbers via trigger. (but maybe misunderstood that part)

Reply:
I did, comments were only ones that were not incremented via triggers.

Reply:
Ahhh got it :)